core/neat/mixins/_grid-column.scss

```diff
@@ -0,0 +1,38 @@
+/// Creates Neat a grid column of requested size.
+///
+/// @argument {number} $width
```

Comment:
Does the number need to be unitless, or include a unit?

Our convention so far has been: `{number (unitless)}` or `{number (with unit)}`